app/assets/stylesheets/application.scss

```diff
@@ -197,6 +200,13 @@ form {
 				background: #f8f8f8;
 			}
 			td {
+				// #viewpost a{
+				// 	color: $orange ;
+				// }
+				.green-text{
+				color: #58d039;}
```

Comment:
Let's add this to variables.scss to make sure we're using the same hex of green throughout the app for future changes.